docs/UserGuide.md

```diff
@@ -410,50 +473,26 @@ Shows a list of all classes in the Class tab.
 
 Format: `listclass`
 
-### Removing students from a class: `removefromclass`
-<hr>
-
-Removes a single or multiple students from an existing class.
-
-Format:
-```
-removefromclass CLASS_INDEX STUDENT_INDEX...
-```
-
-* This command removes any number of existing students from an existing class.
-* `CLASS_INDEX` is the index number of the class in the displayed class list in the `classes` tab, which will be
-  removing the students.
-* `STUDENT_INDEX...` are the index number/s of the students shown in the displayed student list of the class to remove from in the `classes` tab, these students are
-  to be removed from the class.
-
-Example:
-```
-removefromclass 1 1 2 3
-```
-Removes the 1st, 2nd and 3rd student in the displayed student list of the 1st class in the `classes` tab with the 
-`size` decreasing by 3.
-
 ### Deleting a student : `delete`
 <hr>
 
-Deletes the specified student from the address book.
+Deletes the specified student from the TimesTable.
 
 Format:
 ```
 delete INDEX
 ```
 * Deletes the student at the specified `INDEX`.
 * The index refers to the index number shown in the displayed student list.
-* The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd student in the address book.
+* `list` followed by `delete 2` deletes the 2nd student in the TimesTable.
 * `find Betsy` followed by `delete 1` deletes the 1st student in the results of the `find` command.
 
 ### Delete a class: `deleteclass`
 <hr>
 
-Deletes the specified class from the address book.
+Deletes the specified class from the TimesTable.
 
 Format:
 ```
```

Comment:
If we moving `removefromclass` next to `addtoclass`, maybe we should just move `delete` and `deleteclass` next to `add` and `addclass` also.

Reply:
Yeah good catch! Ok